_docs/install.md

```diff
@@ -7,28 +7,33 @@ toc:
 nav: 1
 ---
 
-## Install
+Install
+-------
 
 You can install Pico either using a pre-bundled release or with composer. Pico is also available on [Packagist.org][] and may be included in other projects via `composer require picocms/pico`. Pico requires PHP 5.3+
 
-### Using a pre-bundled release
+### Using a pre-bundled release - for users
```

Comment:
Currently: 


Using a pre-bundled release

__Just download the latest Pico release and upload all files to the httpdocs directory (e.g. /var/www/html) of your server.__

Composer

Step 1 - for users

__Download the source code of Pico's latest release, upload all files to the httpdocs directory (e.g. /var/www/html) of your server and navigate to the upload directory using a shell.__

Step 1 - for developers

Open a shell and navigate to the desired install directory of Pico within the httpdocs directory (e.g. /var/www/html) of your server. You can now clone Pico's Git repository as follows:


Why is the "for users" instructions there twice?